flujo/infra/settings.py

```
@@ -23,6 +23,15 @@
 class Settings(BaseSettings):
     """Application settings loaded from environment variables. Standard names are preferred."""
 
+    # --- Test/CI toggles ---
+    # Exposed so orchestration layers can cheaply short‑circuit expensive persistence
```

Ruff (0.12.2)

Comment contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)